src/styles/components/FaceoffReveal.scss

```diff
@@ -0,0 +1,130 @@
+* {
```

Comment:
Just noticing that all the styles in this stylesheet need to be wrapped in a parent class, such as `.FaceoffReveal` (or whatever your highest-order className is where all these styles should apply). Without this, all the styles here will apply to the whole app.

Reply:
ok cool good to know